testsuite/ui/views/devel/settings/stripe.py

```diff
@@ -11,7 +11,7 @@
 logger = logging.getLogger(__name__)
 
 
-class BillingAddressForm(View):
+class _BillingAddressForm(View):
```

Comment:
I dont think this name change is needed as it requires module-level pylint ignore.

Reply:
Good question. Why? (though it is valid name in my opinion)

Reply:
This one does not require pylint ignore. 
`BillingAddressFroms` is used only by `StripeCCForm` so I assumed that package-private would be correct encapsulation